pages/ObjectDescription.xml

```diff
@@ -114,6 +114,15 @@ taken from <locus target="#6r"/>.
                 <egXML xmlns="http://www.tei-c.org/ns/Examples">
                     <measure unit="leaf">3+69+1</measure>
                 </egXML>
+                <p>Please use only integers in <gi>measure</gi>. For all searching purposes only integers will be used.</p>
+                <p>Instead of placing your measure around the entire thing which you deem a measure, 
+                try a more specific tagging</p>
+                <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                    A + <measure unit="quire">11</measure>
```

Comment:
Should we also state here the total number of quires '12' as in the other example (protection quite + 19 = 20) Or did I misunderstand the example?

Reply:
IMHO, you can do both, what I care about in this passage of GL is that a measure is a measure and not a sentence which contains some measures. would you prefer to dictate what is the actual value which goes here, i.e. either the total inclusive of the protective quires or excluding it? I think I omitted discussing that al together. @thea-m what do you think?

Reply:
I like tagging only the precise measure!  I don't feel qualified to answer the question, though. I'm not sure, if would would have to decide between both options, which would be the wiser choice. I would tend to the inclusive version. Now I would also tend towards developing precise encoding possiblities for both, because the presence of protective quires is quite common (at the same time, it cannot always be known whether a quire is protective, especially from catalogue descriptions. So there should be absolutely room for a bit of flexibility). But I am aware of my tendency to overencode. Sorry for this extremely unhelpful answer.

Reply:
I also think that this should be left to the judgement of the cataloguer/encoder. you would however additionally want to be able to say that your measure of quires includes or not the protective quires. IMHO this is more or less equivalent. let's say we give this a `@type` for which one will the script look? for the inclusive or non inclusive? in either case these may not be both there, so the searches will become inconsistent in that respect and may simply flatten the distinction. if we do not bother distinguishing and selecting as proposed above, we will have an inclusive search for this measure, unaware of inclusiveness or not, and people will find out at the level of the manuscript what was actually given in the measure, that is, searches will be consistent, encoding easy and evidence provided without further complication.

Reply:
That seems reasonable. Do I understand correctly that you propose not introduce any further types but to chose whether we would like to mark up the above example as, for example: 
`A + <measure unit="quire">11</measure>` or  `<measure unit="quire">12</measure> (A+11)`?

Reply:
yes

Reply:
you could also, paradoxically write something like 

> `there is a protective quire (A) which I am not counting in the total of <measure unit="quire">11</measure> quires, but it would make a total of  <measure unit="quire">12</measure> if I did`